14-11-2018/monika/rot13/ror13.rb

```diff
@@ -0,0 +1,31 @@
+class String
+
+  def ror13
+
+    letters = self.split("")
+    polish_letters= ['ą','ę','ó','ś']
+    alphabet = ('a'..'z').to_a
+    alphabet << ('a'..'z').to_a
+    alphabet << ('A'..'Z').to_a
+    alphabet << ('A'..'Z').to_a
+
+    new_alphabet = alphabet.flatten
```

Comment:
this could be constant and/or if you could create this outside of ror13 method, cuz this would be created everytime ror13 is used